examples/v2/sqladmin/jinja/sqladmin.yaml

```diff
@@ -5,4 +5,10 @@ resources:
 - name: database
   type: sqladmin.jinja
   properties:
-    tier: D1
+    tier: db-n1-standard-1
+    region: us-central1
+    readReplicas: 1
+    replicationType: SYNCHRONOUS
+    username: alice
+    password: foo
```

Comment:
Until we have secrets management (a few quarters out still), I don't think we should be encouraging the storage of passwords in our templates. Can you remove the user addition for now? (From both jinja and python.)

Reply:
done